### PR TITLE
feat:administrator blocking user done

### PR DESCRIPTION
**Pull request by SrdjanIlic02**
## Change goal
As an administrator,
I want to review and block user accounts,
So that I can ensure the platform remains safe and functional for all users

Acceptance criteria:
 - The administrator can see the following details when reviewing user accounts: username, email, and role (guide, tourist, administrator).
 - The administrator cannot see the user's password
 - The administrator has the option to block a user account
 - Blocked users are unable to log in to the system
 
 ## UI changes
SS1

### CODE REVIEW

File: Explorer/src/app/feature-modules/administration/model/account.model.ts
Comment (SanjaRadic23) [inline Explorer/src/app/feature-modules/administration/model/account.model.ts:1]: Kod je u redu.

**Review by SanjaRadic23 — APPROVED** [review]

**Comment by Vojnic** [comment]
sve je u redu kod je vise clean

File: Explorer/src/app/feature-modules/administration/account-management/account-management.component.html
Comment (SanjaRadic23) [inline Explorer/src/app/feature-modules/administration/account-management/account-management.component.html:1]: Ispostovane konvencije za znacajne nazive.

File: Explorer/src/app/feature-modules/administration/account-management/account-management.component.ts
Comment (SanjaRadic23) [inline Explorer/src/app/feature-modules/administration/account-management/account-management.component.ts:1]: Komentari bi mogli da budu na engleskom, jer je cela aplikacija na engleskom. Ostatak koda radi sta treba i jasno je definisano sve.

File: Explorer/src/app/feature-modules/administration/administration.service.ts
Comment (SanjaRadic23) [inline Explorer/src/app/feature-modules/administration/administration.service.ts:1]: Kod je u redu.

**Review by SanjaRadic23 — COMMENTED** [review]
Ispostovane su konvencije za znacajne nazive, kod je lepo struktuiran. Mozda ima puno komentara koji nisu toliko bitni.

**Pull request merged**